### PR TITLE
document page created

### PR DESCRIPTION
**Pull request by Horror26**

### CODE REVIEW

File: src/components/About/Documention/Document.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+
+export const Documentation = ({ title, url, img }: DocumentProps) => {
+  return (
+    <div className=''>
+      <a href={url}>
+        <div className=' p-3 gap-1 items-center flex flex-row  border border-solid shadow-sm rounded-lg hover:cursor-pointer '>
+          <div>
+            <img src={img} alt="test" />
```
Comment (JayeshVP24) [inline src/components/About/Documention/Document.tsx:16]: give fixed width

File: src/components/About/Documention/index.astro [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+} from "./data";
+import Document from "./Document.tsx";
+---
+
+<main class="flex flex-col">
+    <div>
+        <p class="font-semibold text-lg text-[#667085] mx-7 mb-5 uppercase">
+            NIRF DATA
+        </p>
+        <div
```
Comment (JayeshVP24) [inline src/components/About/Documention/index.astro:15]: unnecessary two main divs. merge div at line 16 and div at line 18

File: src/components/About/Documention/index.astro [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    InstitutionalActivitesData,
+    EducationalSurveyData,
+} from "./data";
+import Document from "./Document.tsx";
+---
+
+<main class="flex flex-col">
+    <div>
+        <p class="font-semibold text-lg text-[#667085] mx-7 mb-5 uppercase">
+            NIRF DATA
```
Comment (JayeshVP24) [inline src/components/About/Documention/index.astro:13]: make this more dynamic
title and color of one section can also come from array of objects itself.
example 
const data = 
```
const data = [
   {
        title: "NIRF DATA".
        color: "#667085",
        files: [
            {
                title: "sometitle",
                link: "somelink"
            },
            ....
        ]
   },
   .....
]
```

You will need to map twice.

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

File: src/components/About/Documention/index.astro (hunk 3 above)
Comment (JayeshVP24) [reply src/components/About/Documention/index.astro:13]: you will have problems with tailwind colors when doing like this
use inline css with javascript variables for this

File: src/components/About/Documention/Document.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+
+export const Documentation = ({ title, url, img }: DocumentProps) => {
+  return (
+
+    <a href={url}>
+      <div className=' p-3 gap-1 items-center justify-center flex flex-row  border border-solid shadow-sm rounded-lg hover:cursor-pointer '>
+        <div>
+          <img src={img} alt="test" /> </div>
```
Comment (JayeshVP24) [inline src/components/About/Documention/Document.tsx:14]: give this a fixed width, remove give proper alt

File: src/components/About/Documention/index.astro [hunk 5]
```diff
@@ -0,0 +1,74 @@
+---
+import {
+    NIRFData,
+    InstitutionalActivitesData,
+    EducationalSurveyData,
+} from "./data";
+import Document from "./Document.tsx";
+---
+
+<main class="flex flex-col">
```
Comment (JayeshVP24) [inline src/components/About/Documention/index.astro:10]: give `p-4 lg:p-16`
move all this code to pages/about/documentation.astro

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]
last changes

**Review by kamalika0363 — APPROVED** [review]
Approved

**Pull request merged**